superset/views/core.py

```diff
@@ -1036,12 +1036,13 @@ def get_form_data(self, slice_id=None, use_slice_data=False):
             # special treat for since/until and time_range parameter:
             # we need to breakdown time_range into since/until so request parameters
             # has precedence over slice parameters for time fields.
-            if 'time_range' in form_data:
-                form_data['since'], separator, form_data['until'] = \
-                    form_data['time_range'].partition(' : ')
-            if 'time_range' in slice_form_data:
-                slice_form_data['since'], separator, slice_form_data['until'] = \
-                    slice_form_data['time_range'].partition(' : ')
+            if 'since' in form_data:
```

Comment:
`until` should also be in the form_data parameters check list, because request can override `since` or `until` or both